specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2019-01-23-preview/desktopvirtualization.json

```diff
@@ -2277,6 +2170,20 @@
         "ssoContext": {
           "type": "string",
           "description": "Path to keyvault containing ssoContext secret."
+        },
+        "preferredAppGroupType": {

```

Comment:
If this isn't nullable, then it should be required, shouldn't it?

Reply:
adding required